reference-implementation/lib/ReadableStreamBYOBRequest-impl.js

```diff
@@ -14,7 +14,7 @@ exports.implementation = class ReadableStreamBYOBRequestImpl {
       throw new TypeError('This BYOB request has been invalidated');
     }
 
-    if (IsDetachedBuffer(this._view.buffer) === true) {
+    if (CanTransferArrayBuffer(this._view.buffer) === false) {
```

Comment:
We can turn this back into a regular `IsDetachedBuffer` check without problems. `this._view` is always an `Uint8Array` that we constructed ourselves around the buffer of the first pull-into descriptor, which we have been checked earlier (i.e. received from `BYOBReader.read(view)` or created ourselves through `autoAllocateChunkSize`).